main.tf

```diff
@@ -9,28 +9,20 @@ module "label" {
   tags       = "${var.tags}"
 }
 
-module "subnets" {
-  source             = "git::https://github.com/cloudposse/terraform-aws-dynamic-subnets.git?ref=tags/0.2.3"
-  availability_zones = "${var.availability_zones}"
-  namespace          = "${var.namespace}"
-  name               = "${var.name}"
-  stage              = "${var.stage}"
-  region             = "${var.region}"
-  vpc_id             = "${aws_vpc.default.id}"
-  cidr_block         = "${aws_vpc.default.cidr_block}"
-  igw_id             = "${aws_internet_gateway.default.id}"
-  delimiter          = "${var.delimiter}"
-  attributes         = ["${compact(concat(var.attributes, list("subnets")))}"]
-  tags               = "${var.tags}"
-}
-
 resource "aws_vpc" "default" {
-  cidr_block           = "${var.cidr_block}"
-  enable_dns_hostnames = true
-  tags                 = "${module.label.tags}"
+  count                            = "${var.create_vpc ? 1 : 0}"
```

Comment:
This will cause https://github.com/cloudposse/terraform-aws-vpc/pull/10/files#diff-c09d00f135e3672d079ff6e0556d957dR7 to fail.

Reply:
Cannot refer to a resource of `count=0`

Reply:
What's the purpose of introducing this new flag?

Reply:
It works as I'm expect.

true:
<img width="536" alt="screenshot 2017-10-19 14 48 20" src="https://user-images.githubusercontent.com/26582191/31769517-aac8af80-b4dc-11e7-94ba-2a40a5ab086e.png">

false:
<img width="557" alt="screenshot 2017-10-19 14 47 00" src="https://user-images.githubusercontent.com/26582191/31769519-b1fdfee0-b4dc-11e7-90b8-5e92d199c03d.png">

> What's the purpose of introducing this new flag?

I want to use this module in the `terraform-aws-named-subnets`

Reply:
> I want to use this module in the terraform-aws-named-subnets

That's not the correct architecture. VPCs have subnets. Subnets don't have VPCs. Subnets belong to VPCs. Thus in an object-oriented model, we don't want to create VPCs in a subnet module. For example, if you were creating a class to define a "tire", and that tire would be attached to a "car", you wouldn't have the "tire" create a car.